[2024-05-31T16:32:48.537+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-31T16:32:48.560+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v04.group_tasks.transform_data_weather manual__2024-05-31T11:14:41.858816+00:00 [queued]>
[2024-05-31T16:32:48.570+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v04.group_tasks.transform_data_weather manual__2024-05-31T11:14:41.858816+00:00 [queued]>
[2024-05-31T16:32:48.571+0000] {taskinstance.py:2306} INFO - Starting attempt 8 of 13
[2024-05-31T16:32:48.584+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): group_tasks.transform_data_weather> on 2024-05-31 11:14:41.858816+00:00
[2024-05-31T16:32:48.593+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1740) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-31T16:32:48.595+0000] {standard_task_runner.py:63} INFO - Started process 1742 to run task
[2024-05-31T16:32:48.593+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_weather_workflow_DAG_v04', 'group_tasks.transform_data_weather', 'manual__2024-05-31T11:14:41.858816+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpyrub5cqb']
[2024-05-31T16:32:48.596+0000] {standard_task_runner.py:91} INFO - Job 439: Subtask group_tasks.transform_data_weather
[2024-05-31T16:32:48.659+0000] {task_command.py:426} INFO - Running <TaskInstance: my_weather_workflow_DAG_v04.group_tasks.transform_data_weather manual__2024-05-31T11:14:41.858816+00:00 [running]> on host cd9caf1c47f0
[2024-05-31T16:32:48.762+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='my_weather_workflow_DAG_v04' AIRFLOW_CTX_TASK_ID='group_tasks.transform_data_weather' AIRFLOW_CTX_EXECUTION_DATE='2024-05-31T11:14:41.858816+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-31T11:14:41.858816+00:00'
[2024-05-31T16:32:48.763+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-31T16:32:48.794+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-31T16:32:48.795+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-31T16:32:48.813+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_weather_workflow_DAG_v04, task_id=group_tasks.transform_data_weather, run_id=manual__2024-05-31T11:14:41.858816+00:00, execution_date=20240531T111441, start_date=20240531T163248, end_date=20240531T163248
[2024-05-31T16:32:48.850+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-31T16:32:48.897+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-31T16:32:48.901+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
